content/developer/reference/backend/views.rst

```diff
@@ -37,9 +37,10 @@ Fields
 
 View objects expose a number of fields. They are optional unless specified otherwise.
 
-* ``name`` (mandatory) :class:`~odoo.fields.Char`
+* ``name``  :class:`~odoo.fields.Char`
```

Comment:
I think this is a bit misleading: functionally and database-wise, the name _is_ required. However if you do not provide one, it will default to `model_name view_type`